.github/workflows/build-spark.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Databricks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build Spark
```

Comment:
On 'push', we validate that elm file can be compiled to Scala and Scala can be compiled to byte code and packaged as Jar